content/blog/automating-operations-on-dscc-using-ansible-playbooks.md

```diff
@@ -0,0 +1,437 @@
+---
+title: Automating operations on DSCC using Ansible Playbooks
+date: 2023-03-29T14:00:40.866Z
+author: Anusha Y and Sijeesh Kattumunda
+authorimage: /img/Avatar1.svg
+disable: false
+---
+Automation is one of the top trends in technology and the pace of automation is accelerating with more companies opting for developing fully automated systems. Automation reduces time, effort, cost, and manual errors while increasing efficiency and productivity. Gone are those days when many complex coding skills were required to implement automation. Now, there are many low-code tools available in the market, like Ansible, that make automation easier.
+
+In this blog post, I am excited to be able to introduce the Ansible playbooks for DSCC and show you how to use them. Along with the Python SDK for DSCC, these playbooks should help you in your efforts to automate HPE GreenLake Data Services through an infrastructure-as-code approach.
+
+Ansible is an open-source IT automation tool that automates provisioning, configuration management, application deployment, and many other IT processes. 
+Two main features that make Ansible the best choice for automation are:
+
+* Ansible does not require any programming. 
+* Idempotence is offered as a built-in feature of many of the Ansible modules. This means the result of performing a task once is the same as performing it multiple times without any intervening actions.
+
+**Why do we need Ansible playbooks for DSCC?**
+
+Ansible helps the users/admins to automate the deployment of resources and applications without the manual overhead of creating everything from scratch. These playbooks can be configured with conditions, variables, and tasks. Currently, simple playbooks, like performing CRUD operations on the resources, are available. These playbooks can be considered basic building blocks and can be reused to build simple-to-complex use cases.
+
+**Ansible Modules for DSCC:**
+
+The following Ansible modules are currently available for DSCC. You can make use of the samples given or your own Ansible playbooks. Let's look at the Ansible modules available for the DSCC here. These modules are currently available in a GitHub repo.
```

Comment:
These modules are currently available in the GitHub repo [here](put the link here).

Reply:
The github repo will be publicly available within a couple of days. Will add the link then. I've commented out a placeholder on the blog just so I don't miss it.